src/main/java/de/retest/recheck/image/ImageDifference.java

```diff
@@ -4,7 +4,7 @@
 
 public class ImageDifference {
 	// to make sure to account for rounding errors
-	private static final double EQUALITY_THRESHOLD = 0.99;
+	private static final double EQUALITY_THRESHOLD = 0.9999999999;
```

Comment:
is the number chosen arbitrarily?

Reply:
I talked with Martin about that... the number _is_ chosen arbitrarily, but until we have more real-life examples, we probably want to keep it that way... :-/